eve/workers/end2end/scripts/run-e2e-test.sh

```diff
@@ -16,7 +16,16 @@ BACKBEAT_BUCKET_CHECK_TIMEOUT_S=${BACKBEAT_BUCKET_CHECK_TIMEOUT_S:-10}
 POD_NAME="${ZENKO_NAME}-${STAGE}-test"
 TOKEN=$(get_token)
 
+CLOUDSERVER_SECRET_V4=$(kubectl get secret end2end-connector-cloudserver-config.v4 -o jsonpath="{.data.config\.json}}" | base64 -di)
```

Comment:
there should be quotes around the whole `$()` expression

Reply:
there is also a typo in jsonpath expression : 2 braces at the end
```suggestion
CLOUDSERVER_SECRET_V4="$(kubectl get secret end2end-connector-cloudserver-config.v4 -o jsonpath="{.data.config\.json}" | base64 -di)"
```